### PR TITLE
[4.0] [mod_feed] Improvements

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

- Use namespaced classes everywhere

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 36c425cdc8692ffbc75ffe13792671305a21c48a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18939">issues.joomla.org/tracker/joomla-cms/18939</a>.</sub>

File: administrator/modules/mod_feed/Helper/FeedHelper.php [hunk 1]
```diff
@@ -34,18 +37,17 @@ public static function getFeed($params)
 		// Get RSS parsed object
 		try
 		{
-			jimport('joomla.feed.factory');
-			$feed   = new \JFeedFactory;
+			$feed   = new FeedFactory;
 			$rssDoc = $feed->getFeed($rssurl);
 		}
-		catch (\Exception $e)
```
Comment (wilsonge) [inline administrator/modules/mod_feed/Helper/FeedHelper.php]: I think this is a bad change. You need to keep this as global namespace

**Review by wilsonge — CHANGES_REQUESTED** [review]
See comment

**Comment by C-Lodder** [comment]
@wilsonge - Hmm I don't even remember changing that as I know it shouldn't be done :/

Fixed anyway

**Pull request merged**